src/language-server/project/rover/project.ts

```diff
@@ -250,8 +257,17 @@ export class RoverProject extends GraphQLProject {
       this.sendRequest(HoverRequest.type, virtualParams, token),
     );
 
-  onUnhandledRequest: GraphQLProject["onUnhandledRequest"] = (type, params) => {
-    DEBUG && console.info("unhandled request from VSCode", { type, params });
+  onUnhandledRequest: GraphQLProject["onUnhandledRequest"] = async (
```

Comment:
This is in `onUnhandledRequest` since adding listeners in `server.ts` for this feature would be clumsy anyway (it's not a property like `connection.onSemanticTokenRequest`, but `connection.languages.semanticTokens.on` - yes, literally `on` 🤦 

So it seems more readable to do it just here.